app/src/main/index.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+import { app, BrowserWindow } from 'electron'
+import * as path from 'path'
+import { format as formatUrl } from 'url'
+
+const is_development = process.env.NODE_ENV !== 'production'
```

Comment:
When the `NODE_ENV` is set to "production"? If I understand correctly, this is a runtime variable - whose responsibility is to set it? The user?

Reply:
What are the possible options here? 'production' and what else? I would enumerate them in the code in order to keep track of possible options here. In case we will ever want to use it in more complex examples it would already be there. Plus, pls add comment that electron sets it.